### DISCUSSION TITLE
How do I get the Galaxy Brain badge easy?

### DISCUSSION THREAD

**Discussion by aerocetclub** [community/community#139528] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How do I get the Galaxy Brain badge easy?

**Comment by callmesidhu** [comment]
Response to these discussions.

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/139529#discussioncomment-10730190) for reference to why this discussion was closed and locked. Thanks!
